contract/models/res_partner.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-
+# pylint: disable=missing-docstring
```

Comment:
No need to put this in OCA linter

Reply:
It really helps in the editor to only see the real errors. But if there is a way to use to OCA linter in vim I would be glad to hear about it.

Reply:
Yes, but it adds noise for the rest, so please remove it or use another linter. My editor doesn't highlight these things as I pass OCA pylint configuration to it, but I use VSCode.

Reply:
Even using pylint with the pylint_odoo plugin still complains about the missing docstring. So I really think for most people this is supressing a useless message.

Reply:
Then you are going to add this line in all the files you touch since now? This is the same need as the old line
```
# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
that we suffered from OpenERP during several versions.
Please don't do that.
And even more, you are not adding a file, but only a bug fixing that is a one line diff that you are converting to 6-7...